pyproject.toml

```
@@ -20,7 +20,7 @@ classifiers = [
 include = ["topt/py.typed"]
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.8"
```

Python 3.8 is about to hit [end-of-life](https://devguide.python.org/versions/), so let's do
```suggestion
python = "^3.9"
```

---

Done. :+1: